CHANGELOG.md

```diff
@@ -7,18 +7,25 @@ and this project uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
-### Changed
+### Added
 - `search_datasets` now accepts a `has_granules` keyword argument. Use
   `has_granules=False` to search for metadata about collections with no
   associated granules. The default value set in `DataCollections` remains `True`.
   ([#939](https://github.com/nsidc/earthaccess/issues/939))
   ([**@juliacollins**](https://github.com/juliacollins))
 
+### Changed
+- **Breaking**: Throw an error by default when login credentials are rejected. Pass
+  `earthaccess.login(..., raise_on_failure=False)` to disable and print a warning
+  instead. ([#946](https://github.com/nsidc/earthaccess/pull/946))
+  ([**@mfisher87**](https://github.com/mfisher87))
+
 ## [v0.13.0] - 2025-01-28
 
 ### Changed
-- Integration tests: Test are no longer randomized! this means each fail should be reproducible, we are testing the most
-  popular datasets from all DAACs, see files under tests/integration/popular_collections.
+- Integration tests: Test are no longer randomized! this means each fail should be
+  reproducible, we are testing the most popular datasets from all DAACs, see files under
+  tests/integration/popular_collections.
```

Comment:
I thought we decided to exclude non-user-facing changes from the changelog.

Reply:
Since you touched it, might as well just remove it, but not bothered if you leave it.

Reply:
OK I'll yank it since I want it gone anyway :)

Reply:
faa295b

Reply:
5539a25

Reply:
I personally am on the other side of the house; I think changes that are useful for developers *or* users to know should be included as they are both "consumers" of earthaccess, and when I'm working on the code, the CHANGELOG is a lot closer than the releases.

But Common Chagelog defines it as "distributed form" not "repository" so, :shrug: . 

We did discuss it before, but I don't remember a definitive resolution. Maybe we should put a sentence at the top of the changelog about it?